### DISCUSSION TITLE
Authenticated express routes

### DISCUSSION THREAD

**Discussion by borg** [keystonejs/keystone#6577] [Event Q&A Q3 2021]
The newly released express config option is neat, but it KS5 the RestApp could have access to both the general authenticated session context, and access db in "god mode" - what's the equivalent for KS6? (I know I can highjack the session cookie, but that feel hackyish)

**Comment by bladey** [comment]
Hey @borg,

Thanks for your question!

Would you mind posting this question to our community Slack with a few more details on what you refer to as the `RestApp`, and a link to the equivalent docs for Keystone 5?

That's the best place to get focused support on blocks you encounter with Keystone - https://community.keystonejs.com
